benchmarks/transactions-generator/Cargo.toml

```diff
@@ -9,21 +9,13 @@ license.workspace = true
 publish = false
 
 [features]
-nightly_protocol = [
-  "near-async/nightly_protocol",
-  "near-client-primitives/nightly_protocol",
-  "near-client/nightly_protocol",
-  "near-network/nightly_protocol",
-  "near-primitives/nightly_protocol",
-  "node-runtime/nightly_protocol",
-]
 nightly = [
   "near-async/nightly",
   "near-client-primitives/nightly",
   "near-client/nightly",
   "near-network/nightly",
   "near-primitives/nightly",
-  "nightly_protocol",
+
```

Comment:
Remove extra line where `nightly_protocol` used to be. Do this for all toml files modified

Reply:
Remove new line in all toml files modified